jupyterlab_caip_optimizer/src/components/snackbar_widget.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { Snackbar as MaterialSnackbar } from '@material-ui/core';
+import { Alert as MuiAlert, AlertProps } from '@material-ui/lab';
```

Comment:
Assuming the Mui stands for MaterialUi, maybe we could rename it to MaterialAlert (similar to how MaterialSnackbar is renamed)?